src/types/formTypes.ts

```diff
@@ -338,8 +343,8 @@ export const pointOfContactSchema = z.object({
     }),
   phoneExtension: z
     .string()
-    .max(inputCharLimit, {
-      message: "The phone number extension's maximum character limit is 255",
+    .max(phoneExtensionNumberLimit, {
+      message: PocZodSchemaErrors.phoneExtensionMin,
```

Comment:
Consider renaming **PocZodSchemaErrors.phoneExtensionMin** to **PocZodSchemaErrors.phoneExtension** since it is used for more than the min value validation.